src/main/java/jenkins/plugins/office365connector/utils/Parser.java

```diff
@@ -0,0 +1,43 @@
+package jenkins.plugins.office365connector.utils;
+
+import hudson.model.AbstractBuild;
+import hudson.model.Cause;
+import jenkins.plugins.office365connector.model.Fact;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Created by olegfeferman on 23.11.2017.
```

Comment:
we can already find this information in git history - add more information or delete it at all

Reply:
automatically added by code editor but yepp, can delete that